commands/registry.go

```diff
@@ -0,0 +1,224 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/doctl/do"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+	k8sapiv1 "k8s.io/api/core/v1"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sjson "k8s.io/apimachinery/pkg/runtime/serializer/json"
+)
+
+type dockerConfig struct {
+	Auths map[string]struct {
+		Auth string `json:"auth,omitempty"`
+	} `json:"auths"`
+}
+
+// Registry creates the registry command
+func Registry() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "registry",
+			Aliases: []string{"reg", "r"},
+			Short:   "registry commands",
+			Long:    "registry is used to access container registry commands",
```

Comment:
As this is being released as a private beta which requires the user to first sign up and then be invited, can we label the descriptions with `[Beta]`? Additionally, it might make sense to hide this from menus at this point.

Reply:
+1 on the hide